analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.cs

```diff
@@ -216,6 +217,27 @@ public void Execute_CheckProducesMoreStates_Both()
                 .And.ContainSingle(x => x.HasConstraint(TestConstraint.Second) && x.HasConstraint(BoolConstraint.False));
         }
 
+        [TestMethod]
+        public void Execute_ClearsCapturesAfterBranching()
+        {
+            var allReferences = new List<IOperation>();
+            var collector = new PostProcessTestCheck(x =>
+            {
+                if (x.Operation.Instance.Kind == OperationKind.FlowCaptureReference)
+                {
+                    allReferences.Add(x.Operation.Instance);
+                }
+                return x.State;
+            });
+            var validator = SETestContext.CreateCS(@"string a = null; a ??= arg; Tag(""End"");", ", string arg", collector).Validator;
```

Comment:
The snippet would be more easily readable if the lines would be one under the other.